api/soap/mantisconnect.wsdl

```diff
@@ -259,6 +266,43 @@
    <xsd:element name="url" type="xsd:string" minOccurs="0"/>
   </xsd:all>
  </xsd:complexType>
+ <xsd:complexType name="FilterSearchData">
+  <xsd:all>
+   <xsd:element name="project_id" type="tns:IntegerArray" minOccurs="1"/>
```

Comment:
I don't think we should make fields like project_id, issues_per_page and page_number required.  Each one has a sensible default value: project_id should default to 0 (for all projects), issues_per_project to the configuration option that determines number of issues on View Issues page, page number to first page (I assume we are 1-based).